db/change_listener.go

```diff
@@ -95,8 +95,8 @@ func (listener *changeListener) ProcessFeedEvent(event sgbucket.FeedEvent) bool
 	requiresCheckpointPersistence := true
 	if event.Opcode == sgbucket.FeedOpMutation || event.Opcode == sgbucket.FeedOpDeletion {
 		key := string(event.Key)
-		if strings.HasPrefix(key, auth.UserKeyPrefix) ||
-			strings.HasPrefix(key, auth.RoleKeyPrefix) { // SG users and roles
+		if (strings.HasPrefix(key, auth.UserKeyPrefix) ||
+			strings.HasPrefix(key, auth.RoleKeyPrefix)) && event.Opcode == sgbucket.FeedOpMutation { // SG users and roles
```

Comment:
We still need to do the notify on line 103 in the case of a delete mutation (to trigger active changes feeds to attempt to reload the user, then terminate when the user isn't found).  

I think you can just move the `&& event.Opcode == sgbucket.FeedOpMutation` down to line 100.